### ISSUE TITLE
Passing elements to JS

### ISSUE THREAD

**Issue by tuvistavie** [HashNuke/hound#84]
Hi,

I was trying to pass an element to JS and got some issue.
I would expect

```elixir
element = find_element(:id, "foo")
execute_script("console.log(arguments[0])", [element])
```

to pass the element to JS, as most selenium implementation do,
however, it passes the ID, which is not very useful.

After digging a little in other implementations, I found out that it must be sent as

```
{"ELEMENT": "ELEMENT_ID"}
```

for it to work, so

```elixir
element = find_element(:id, "foo")
execute_script("console.log(arguments[0])", [%{"ELEMENT" => element}])
```

does the job correctly.

However, I think it would be neater if we could do this automatically.

One solution could be to stop working with ids as strings directly, but
maybe wrap them in a struct.
This would make handling this kind of case trivial, as we would only
need to define an implementation of `Poison.Encoder` to get things to work.

What do you think?

**Comment by HashNuke** [comment]
@tuvistavie yes it makes sense. I never looked at this use case before. Please do send a PR if possible.

**Comment by tuvistavie** [comment]
@HashNuke Sure, I am going to give it a try and I'll send a PR.

**Comment by HashNuke** [comment]
@tuvistavie thank you. Looking forward to it (whenever you find the time).

**Comment by tuvistavie** [comment]
Fixed by #87 !
